data/questions.js

```diff
@@ -0,0 +1,44 @@
+export const questions = [
+  {
+    "id": 1,
+    "question": "Explain what ARIA and screenreaders are, and how to make a website accessible.",
+    "answers": [],
+  }, {
+    "id": 2,
+    "question": "What is the virtual DOM in React?",
+    "answers": [],
```

Comment:
Nice job, adding a comma at the end. These are good developer empathy habits.

Reply:
Thanks man, I try!